### ISSUE TITLE
[Bug] Jail Interior Gates

### ISSUE THREAD

**Issue by Trauma-Llama** [FrenzyRoleplay/server-issues#614]
**Describe the bug**
There are times that the jail cell gates will not be closed for people. This is a problem when we toss people into solitary, uncuff them in the cell and they just run out because in "their eyes" they do not see the gate being closed. I want to assume it is a desync issue. 

**To Reproduce**
This is hard to reproduce as you may not see it happen in your eyes.
head into the prison and check the gates with another person and see if you can get one more person to not see gates closed.
Could also be that locking the gate while it is open and it closing doesn't happen in their eyes.
basically a Trial and error till till it happens.
**Expected behavior**
Doors/gates of jail cells will be closed in everyones eyes

**Screenshots**

**Additional context**
I do believe this may be a desync or a Lock issue on the gates, I'm not sure.
May be an issue with the locks, If we lock it while it is still open and then let it close, it may not be fully closing in their eyes.
